wdfn-server/waterdata/views.py

```diff
@@ -170,6 +161,26 @@ def return_404(*args, **kwargs):
     return abort(404)
 
 
+def get_cooperator_data(location_with_values, site_no):
```

Comment:
I would prefer that this go into utils. We should only

Reply:
Good idea